lib/run-script.js

```diff
@@ -125,9 +125,16 @@ function run (pkg, wd, cmd, args, cb) {
   } else {
     if (!pkg.scripts[cmd]) {
       if (cmd === "test") {
-        pkg.scripts.test = "echo \"Error: no test specified\"";
+        pkg.scripts.test = "echo \"Error: no test specified\""
+      } else if (cmd === "env") {
+        log.verbose("run-script using default multiplatform env")
```

Comment:
that is trick... `set` doesn't necessarily have arguments either. if the user is on solaris csh, it won't understand any of the options set from bash understands.
I'm honestly not sure what most use cases would expect.... for me, since all argument i see are for changing future behavior of the shell, i think nobody will have reasons to pass arguments on a build from this command. all i can see this being used is to probe the vars npm will add when running the script the use is writing/debugging.